test/metabase/query_processor_test/test_mlv2.clj

```diff
@@ -120,7 +116,7 @@
      {:aggregation aggregations}
      (mbql.u/match-one aggregations
        :metric
-       "#28689"))
+       "#29936"))
```

Comment:
This was the wrong issue number; `#28689` is the epic, I meant to write `#29936` originally.